js/Paola-Fatima.js

```diff
@@ -0,0 +1,30 @@
+const menu = {
+  pepperoni: {
+    toppings:['garlic', 'tomatoes', 'pepperoni'],
+    price:"4"
+  },
+  cheese: {
+  toppings: ['spinach', 'mushrooms', 'pineapple'],
+  price:"$5"
+  },
+  margarita: {
+    toppings: ['onions', 'feta', 'sausage'],
+    price: "$6"
+  },
+  everything: {
+    toppings: ['sauce', 'oregano', 'ranch'],
+    price: "$7"
+  },
+  mediterrean: {
+    toppings: ['yogurt', 'vinegar', 'alfredo'],
+    price: "$8"
+  }
+}
+
+console.log(Object.keys(menu))
+
+if ('banana' in menu) {
+  console.log("true")
+} else {
+  console.log("False")
+}
```

Comment:
nice! love to see people using classic tools instead of fancy methods. 🥳 🎈 💯 

in case you're curious, though, there is an object method called [`hasOwnProperty()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty)